src/main/resources/application.yml

```diff
@@ -85,6 +85,7 @@ staff-notifications:
 
 feature_toggles:
   watermark_pdf: false
+  emailToStaff: fasle
```

Comment:
There is no such thing as `fasle` ;)

Reply:
Other thing is that notifications should be enabled by default.

Reply:
Good spot! 
I was testing that it was working properly and swapped it from true to see if it worked. It will be changed in the next commit